opentelemetry-prometheus/tests/integration_test.rs

```diff
@@ -15,6 +15,7 @@ use opentelemetry_sdk::Resource;
 use opentelemetry_semantic_conventions::resource::{SERVICE_NAME, TELEMETRY_SDK_VERSION};
 use prometheus::{Encoder, TextEncoder};
 
+#[ignore]
```

Comment:
please point to the issue or the PRs trying to fix this for easy referenc.e

Reply:
I add the pr link